src/main/java/dev/jbang/cli/BaseCommand.java

```diff
@@ -61,19 +64,13 @@ boolean isVerbose() {
 		return Util.isVerbose();
 	}
 
+	boolean isQuiet() {
+		return Util.isQuiet();
+	}
+
 	@Override
 	public Integer call() throws IOException {
-		try {
-			return doCall();
-		} catch (ExitException e) {
```

Comment:
where is this handled if removed from here?

Reply:
In the ExitHandler, it contained the exact same code

Reply:
ah true...